### ISSUE TITLE
PVC property in linstor volume/resource definition

### ISSUE THREAD

**Issue by phoenix-bjoern** [piraeusdatastore/piraeus-operator#163]
For debugging we often have to check the status of the volume/resource in the linstor CLI. Therefore we first have to check the PV name the CSI driver has generated for the PVC. With the PV name we can to execute `linstor v l|grep <pv_name>` to see what the actual status of the volume/resource in DRBD is.

To ease that process it would be helpful to write the PVC name in a custom property of the volume and/or resource. In addition then linstor command should be extended to add a column for additional properties specified in an argument (but this feature request has to be raised at https://github.com/LINBIT/linstor-server).
With that you could easily find out which ResourceName (PV name) and DeviceName belong to the PVC name, which significantly will speed up analysis.

Is that a feature of general interest?

**Comment by WanzenBug** [comment]
Yes, it's something that's been requested a few times. What would need to happen:
* enable the `--extra-create-metadata` option on the csi-provisioner
* update linstor-csi to support the extra parameters passed in
The main question is how this information is then passed to LINSTOR. I believe there is already a "ExternalName" field that could support arbitrary strings for names.
